pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/index/metadata/SegmentMetadataImpl.java

```diff
@@ -192,6 +193,13 @@ private void loadCreationMeta(File crcFile)
       final DataInputStream ds = new DataInputStream(new FileInputStream(crcFile));
       _crc = ds.readLong();
       _creationTime = ds.readLong();
+      try {
+        _dataCrc = ds.readLong();
+      } catch (IOException e) {
+        LOGGER.debug("Could not find data crc, falling back to default LONG_MIN value");
+      } finally {
+        ds.close();
```

Comment:
No need to close it here. We are not throwing the exception.
A better approach would be to trap the `DataInputStream` construct into a try-with-resource block to ensure it is always closed